scripts/module/hyper.nut

```diff
@@ -2,54 +2,64 @@
 
 function onScriptLoad() {
 
-  HYPER_DOWN  <- BindKey( true, 69, 0, 0 );//H
-  HYPER_UP    <- BindKey( true, 81, 0, 0 );//F
-  HYPER_FORWARD  <- BindKey( true, 73, 0, 0 );//|
-  HYPER_BACK <- BindKey( true, 75, 0, 0 );//|
-  HYPER_LEFT  <- BindKey( true, 74, 0, 0 );//<
-  HYPER_RIGHT <- BindKey( true, 76, 0, 0 );//>
-  hyper <- array( 100, false );
+  HYPER_DOWN    <- BindKey( true, 0x45, 0, 0 ); // E
+  HYPER_UP      <- BindKey( true, 0x51, 0, 0 ); // Q
+  HYPER_FORWARD <- BindKey( true, 0x49, 0, 0 ); // I
+  HYPER_BACK    <- BindKey( true, 0x4B, 0, 0 ); // K
+  HYPER_LEFT    <- BindKey( true, 0x4A, 0, 0 ); // J
+  HYPER_RIGHT   <- BindKey( true, 0x4C, 0, 0 ); // L
 
+  hyper <- array( 100, false );
 }
 
 
 function onKeyDown(player, id) {
-  
-    
-    if ( hyper[ player.ID ] )
-    {
-        if (id == HYPER_UP) player.Pos.z += 15;
-        else if (id == HYPER_DOWN) player.Pos.z -= 15;
-        else 
-        {
-            local rad = player.Angle, pos = player.Pos;
-            if (id == HYPER_FORWARD);
-            else if (id == HYPER_BACK) rad -= 3.14;
-            else if (id == HYPER_LEFT) rad += 1.570795;
-            else if (id == HYPER_RIGHT) rad -= 1.570795; 
-            else return 0;
-            local x = pos.x, y = pos.y;
-            local x2 = x + 1.0 * cos(rad) - 15.0 * sin(rad);
-            local y2 = y + 1.0 * sin(rad) + 15.0 * cos(rad);
-            player.Pos = Vector( x2, y2, pos.z );
-        }
+  if ( hyper[ player.ID ] ) {
+    if (id == HYPER_UP) {
+      player.Pos.z += 15;
+    }
+    else if (id == HYPER_DOWN) {
+      player.Pos.z -= 15;
+    }
+    else {
+      local rad = player.Angle, pos = player.Pos;
+      if (id == HYPER_FORWARD);
+      else if (id == HYPER_BACK) {
+	rad -= 3.14;
+      }
+      else if (id == HYPER_LEFT) {
```

Comment:
I prefer `} else` on the same line; all other instances too

Reply:
I told you I'd convert to `stroustrup` style, he does not use trust in the OTB.

Reply:
Who or what is OTB?

I personally think that newline before an `else` is a sin. The newline is quickly interpreted as: "this statement ends here", but then it surprisingly continues. It's *really* bad in my opinion.

It also makes it way too easy to accidentally re-arrange codeblocks to cause issues.
It's similar to the apple `goto fail` mess.

I don't care about style-names or whatever. I care about code being secure / predictable, and learning from past mistakes.